src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -588,91 +648,70 @@ NOTE: More details about the source code of this task can be found https://panda
 
-==== Feature Extraction
-
-===== Feature_Vector_Extractor
+===== Filter_Data
 
-*Task Overview:* Encode *structured* data into numerical feature vectors whereby machine learning models can be applied.
+*Task Overview:* Query the columns of your data with a boolean expression.
 
 *Task Variables:*
 
-.Feature_Vector_Extractor_Task variables
+.Filter_Data_Task variables
 [cols="2,5,2"]
 |===
 | *Variable name* | *Description* | *Type*
-| `SESSION_COLUMN`
-| the ID of the entity that you need to represent (to group by).
+| `QUERY`
+| The query string to evaluate.
 | String
-| `FILE_OUT_FEATURES`
-| The extension of the file where the resulted features will be saved.
+| `FILTERED_FILE_OUPUT`
+| Refers to the extension of the file where the resulted filtered data will be saved.
 | String [CSV or HTML]
-| `PATTERN_COLUMN`
-| The index of column containing the log patterns.[specific to features extraction from logs]
-| String
-| `PATTERNS_COUNT_FEATURES`
-| True if you need to extract count the number of occurrence of each pattern per session.
-| Boolean [True or False]
-| `STATE_VARIABLES`
-| The different variables that need to be considered to extract features according to their content.
+|===
 
-N.B: separate the different variables with a comma ','
-| String
-| `COUNT_VARIABLES`
-| Refers to the different variables that need to be considered to count their distinct content.
-| String
+NOTE: More details about the source code of this task can be found https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.query.html?highlight=query[here^].
 
-N.B: separate the different variables with a comma ','
-| `STATE_COUNT_FEATURES_VARIABLES`
-| True if you nedd to extract state and count features per session.
-| Boolean [True or False]
+==== Feature Extraction
+
+===== Summarize_Data
+
+*Task Overview:* Calculate the histogram of a dataframe based on a reference column that need to be specified in the variable `REF_COLUMN`.
+
+*Task Variables:*
+
+.Summarize_Data_Task variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `GLOBAL_MODEL_TYPE`
+| The model that will be used to summarize grouped by data.
+| List [KMeans, PolynomialFeatures] (default=KMeans)
+| `REF_COLUMN`
+| The column that will be used to group by the different histogram measures.
+| String
 |===
 
-*How to use this task:* Could be connected with <<Add_Label>> if you need to train a model using supervised learning algorithms or with <<Train_Clustering_Model>> if you need to train a model using unsupervised machine learning techniques.
+NOTE: More details about the source code of this task can be found https://docs.scipy.org/doc/numpy/reference/generated/numpy.histogram.html[here^].
 
-===== Time_Series_Feature_Extractor
+===== Tsfresh_Features_Extraction
 
-*Task Overview:* Extract features by considering the temporal distribution of data.
+*Task Overview:* Calculate a comprehensive number of time series features based on the library TSFRESH.
```

Comment:
Suggestion: add a link to the library

Reply:
For the link of the TSFRESH library it already exists in the Note sub section. For all the other comments, they are fixed now. Thanks for the review 👍